### ISSUE TITLE
Parameter "out_dir" is not used

### ISSUE THREAD

**Issue by tsalo** [62442katieb/idconn-retrieval#10]
`out_dir` is an argument in the parser and in the main workflow function, but it's never actually used. The easiest solution might be to drop it completely.

**Comment by 62442katieb** [comment]
Yeah, it predates the switch to all BIDS all the time and I kept it around just in case I ended up needing it later. I think you're right, though. I'll put it on my list.
